Cargo.toml

```diff
@@ -19,20 +19,20 @@ travis-ci = { repository = "PyO3/pyo3", branch = "master" }
 appveyor = { repository = "fafhrd91/pyo3" }
 
 [dependencies]
-indoc = "0.3.4"
-inventory = "0.1.4"
-libc = "0.2.62"
-num-bigint = { version = ">= 0.2", optional = true }
-num-complex = { version = ">= 0.2", optional = true }
-num-traits = "0.2.8"
+indoc = "^0.3.4"
```

Comment:
`^` is the default operator, this doesn't change anything.

Reply:
Thanks, I misunderstood that it is equal to "=="